tests/testthat/test-cohort_methods.R

```diff
@@ -831,6 +831,19 @@ test_that("Defining and accessing description works fine", {
   )
 })
 
+test_that("steps_range from Step.R returns empty character when from is greater than to", {
+  expect_equal(steps_range(3,2),character(0))
+})
+
+test_that("eval_step_filters from Step.R returns empty character when step id is equal", {
```

Comment:
We should not specify file name, usually using function name is enoght.

```suggestion
test_that("eval_step_filters returns empty character when step id is equal", {
```